lib/seam/clients/connect_webviews.rb

```diff
@@ -23,13 +23,25 @@ def list
         )
       end
 
-      def create(accepted_providers: nil)
+      def create(
+        accepted_providers: [],
```

Comment:
With the new `provider_category` option, this is now optional as well.

```suggestion
        accepted_providers: nil,
```

Reply:
Good call I will have a look. Previously specifying `nil` explicitly was giving me an error - but since I added the `compact` which removes keys for nil values it should probably not be an issue anymore.

Reply:
Done! :)